src/types/index.ts

```diff
@@ -19,6 +19,7 @@ export type LocaleOptions =
   | 'de-DE'
   | 'en-US'
   | 'es-ES'
+  | 'et-ET'
```

Comment:
From what I've Googled it, the ISO code for Estonian is `et-EE`.

Can you please confirm and change it if that's the case?
PS: In case that's correct, please change on the `stories/data.ts` too.

Reply:
You're right about that. Fixed it.